### PR TITLE
scripts.quickstart: Use environment variables for executables

### PR DESCRIPTION
**Pull request by f9n**

### CODE REVIEW

**Comment by brancz** [comment]
I think we’re going a little abstract here. This is just a QuickStart script, I feel we’re at a point where if we introduce these things it’ll grow less useful for the QuickStart purpose and become a script people start using for prod deploys which we absolutely do not want people to do.

File: scripts/quickstart.sh [hunk 1]
```diff
@@ -5,6 +5,11 @@
 
 trap 'kill 0' SIGTERM
 
+export MINIO_EXECUTABLE="minio"
```
Comment (bwplotka) [inline scripts/quickstart.sh]: Why not just variable without `export`?

Comment (f9n) [reply scripts/quickstart.sh]: I just want to use variables dynamically. I didn't care to define it as an environment variable.

Comment (bwplotka) [reply scripts/quickstart.sh]: So can we remove export then? (:

Comment (f9n) [reply scripts/quickstart.sh]: Sure :)

File: scripts/quickstart.sh [hunk 2]
```diff
@@ -168,7 +175,7 @@ sleep 0.5
 # Start to query nodes.
 for i in `seq 1 2`
 do
-  ./thanos query \
+  $THANOS_EXECUTABLE query \
```
Comment (bwplotka) [inline scripts/quickstart.sh]: One more thing: Technically `${...}` is the pattern for variables. Can we be consistent here and on other variables you define here?

**Pull request merged**